### PR TITLE
FEATURE: TMA-1588 Bump brick version to 3.7.10

### PR DESCRIPTION
**Pull request by phong-nguyen-duy**

### CODE REVIEW

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/160/ : SUCCESS in 7s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-lcm-build-pipeline/340/ : SUCCESS in 20m 47s

File: lib/gooddata/lcm/actions/synchronize_ldm.rb [hunk 1]
```diff
@@ -44,6 +44,9 @@ class SynchronizeLdm < BaseAction
 
         description 'Specifies how to synchronize LDM and resolve possible conflicts'
         param :synchronize_ldm, instance_of(Type::SynchronizeLDM), required: false, default: 'diff_against_master_with_fallback'
+
+        description 'Allows to have include deprecated with model diff/view'
```
Comment (danh-ung) [inline lib/gooddata/lcm/actions/synchronize_ldm.rb]: please update description: Enables handling of deprecated objects in the logical data model.

Comment (phong-nguyen-duy) [reply lib/gooddata/lcm/actions/synchronize_ldm.rb]: Updated

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/164/ : SUCCESS in 11s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-lcm-build-pipeline/341/ : SUCCESS in 22m 26s

**Comment by phong-nguyen-duy** [comment]
ok to test

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/165/ : SUCCESS in 6s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-lcm-build-pipeline/342/ : SUCCESS in 30m 19s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/175/ : SUCCESS in 6s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-lcm-build-pipeline/343/ : SUCCESS in 22m 50s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-lcm-promote-pipeline/126/ : SUCCESS in 23m 04s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-merge-compliance-check/409/ : SUCCESS in 18s
- https://checklist.intgdc.com/job/lcm/job/gooddata-ruby-travis-check/83/ : SUCCESS in 5s

**Pull request merged**